src/modules/dashboard/lists/ListItemWrapper.tsx

```diff
@@ -54,6 +54,7 @@ export const ListItemWrapper = ({
   showExternalIncentivesTooltips = {
     superFestRewards: false,
     spkAirdrop: false,
+    kernelPoints: false,
   },
   ...rest
 }: ListItemWrapperProps) => {
```

Comment:
Why don't you put the <KernelAirdropTooltip> here?
Instead of implement it in each SupplyItem, SupplyPosition... components